swift-sdk/Internal/in-app/InAppManager.swift

```diff
@@ -316,6 +323,10 @@ class InAppManager: NSObject, IterableInternalInAppManagerProtocol {
                               callback: ITBURLCallback? = nil) {
         ITBInfo()
         
+        if message.isJsonOnly {
+            return
+        }
+        
```

Comment:
Just personal preference to keep up with the style of the rest of the func.
```suggestion
        guard !message.isJsonOnly else {
            return
        }
        
```